scripts/write-configs.js

```diff
@@ -0,0 +1,106 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+const jsYaml = require('js-yaml');
+const fs = require('fs');
+const path = require('path');
+const tsNode = require('ts-node');
+
+tsNode.register();
+
+const kangarooConfig = require(path.join(process.cwd(), 'kangaroo.config.ts')).default;
+
+if (!process.env.KANGAROO_DEV) {
+  // CHECK THAT NO DEFAULT VALUES ANYMORE
+  if (kangarooConfig.appId === 'org.holochain.kangaroo-electron')
```

Comment:
I might pull out these default config values into a separate file that is imported here and in `kangaroo.config.ts`, or else define them as consts at the top of the file to make super clear they're config

Reply:
I see that we already have some of them in `const.ts`, so maybe make that the source of truth here.

Reply:
They're not runtime constants though so I don't think adding them to `const.ts` makes sense. But maybe adding them to the top of the file for clarity makes sense.

Reply:
3f141e68092cdc64b04b44d42e505d252ed5ae3c